launch/offline_nerian_warthog.launch.yaml

```diff
@@ -0,0 +1,44 @@
+## Online Lidar VTR3
+session_name: vtr_online_nerian_warthog
+
+environment:
+  CYCLONEDDS_URI: ${VTRSRC}/config/cyclonedds.default.xml # custom dds configs
```

Comment:
This is unused and can be deleted. We don't use cyclone anymore.

Reply:
nerian is not used?